src/main/java/ch/astorm/jchess/util/UnicodePositionRenderer.java

```diff
@@ -0,0 +1,73 @@
+package ch.astorm.jchess.util;
+
+import ch.astorm.jchess.core.Color;
+import ch.astorm.jchess.core.Moveable;
+import ch.astorm.jchess.core.Position;
+import ch.astorm.jchess.core.entities.*;
+
+import java.io.PrintStream;
+
+public class UnicodePositionRenderer extends AbstractTextPositionRenderer {
+    private final  String lineSeparator = super.getLineSeparator();
+    private final String emptyDarkCell;
+    private final String emptyLightCell;
+
+    /**
+     * Creates a renderer to the specified {@code out} stream.
+     */
+    private UnicodePositionRenderer(PrintStream out, String emptyDarkCell, String emptyLightCell) {
+        super(out);
+        this.emptyDarkCell = emptyDarkCell;
+        this.emptyLightCell = emptyLightCell;
+    }
+
+    public static void render(PrintStream out, Position position) {
+        render(out, position, " ", " ");
+    }
+
+    public static void render(PrintStream out, Position position, String emptyDarkCell, String emptyLightCell) {
+        new UnicodePositionRenderer(out, emptyDarkCell, emptyLightCell).render(position);
+    }
+
+    @Override
+    public CharSequence renderToString(Position position) {
+        StringBuilder sb = new StringBuilder(182);
+        int nbRows = position.getBoard().getRowsCount();
+        int nbColumns = position.getBoard().getColumnsCount();
+        for (var i=nbRows-1; i>=0; i--) {
```

Comment:
I didn't see it before: the white pieces should be displayed on the bottom and the black pieces at the top to be consistent with the ASCII renderer.

Reply:
Pieces are displayed correctly (Sadly "whites" in unicode are transparent, and "black" are filled)
The numbering was backward and has been corrected